### ISSUE TITLE
Get all objects?

### ISSUE THREAD

**Issue by vtjiles** [aschuch/AwesomeCache#59]
Is there a way to grab all objects from the cache or can they only be accessed by key?

**Comment by dluksza** [comment]
Cache class has a private method allKeys() that returns list of available keys in cache. Simple solution for this problem would be to make it public

**Comment by vtjiles** [comment]
Thanks what I figured. Had tried working around it in an extension to no avail.

**Comment by aschuch** [comment]
Yes there is no way to get them all at the moment. Thanks for the suggestion, I'll keep it in mind for v3.0.

**Comment by dluksza** [comment]
@aschuch do you have any ideas about the release date of 3.0?

**Comment by aschuch** [comment]
You can already have a look at the working `three` branch to check it out. I'll try to release 3.0 as soon as I am satisfied with the API.

**Comment by sasvarizoli** [comment]
I already created a pull request (#62).
